client/src/components/feedback/feedback.js

```diff
@@ -0,0 +1,144 @@
+          allFeedback: data,
+          pieceFeedback: data.slice(0, maxCommentShow),
+        };
+      }
+      return {
+        allFeedback: data,
+        pieceFeedback: data.slice(0, last + 1),
+      };
+    });
+  }
+
+  onClickShowMoreComment() {
+    const {
+      firstLoad, allFeedback, last, maxCommentShow,
+    } = this.state;
+    if (!firstLoad) {
+      this.setState({
+        last: last + maxCommentShow,
+        firstLoad: true,
+      });
+    }
+    if (allFeedback.length >= last + maxCommentShow) {
+      this.setState((prevState) => {
+        return {
+          pieceFeedback: prevState.allFeedback.slice(prevState.start,
+            prevState.last + prevState.maxCommentShow),
+          last: prevState.last + prevState.maxCommentShow,
+        };
+      });
+    } else if (allFeedback.length - last > 0) {
+      this.setState((prevState) => {
+        return {
+          pieceFeedback: prevState.allFeedback.slice(prevState.start,
+            prevState.last + prevState.maxCommentShow),
+          last: allFeedback.length,
+        };
+      });
+    }
+  }
+
+  onClickShowLessComment() {
+    const { maxCommentShow } = this.state;
+    this.setState((prevState) => {
+      if (prevState.last - prevState.maxCommentShow > maxCommentShow) {
+        return {
+          pieceFeedback: prevState.allFeedback.slice(prevState.start,
+            prevState.last - prevState.maxCommentShow),
+          last: prevState.last - prevState.maxCommentShow,
+        };
+      }
+      return {
+        last: 5,
+        pieceFeedback: prevState.allFeedback.slice(prevState.start, 5),
+      };
+    });
+  }
+
+  refresh() {
+    this.componentDidMount();
+  }
+
+  render() {
+    const stateFeed = this.state;
+    return (
+      <Container className="feedback">
+        <FeedbackSend id={this.props.id} refreshed={this.refresh} />
+        {
+          stateFeed.pieceFeedback.map((item) => {
+            return (
+              <FeedbackComment item={item} key={item.id} />
+            );
+          })
+        }
+        <div className="showComments">
+          {this.state.last > 5
+          && (
+          <span
+            className="commSpan"
+            tabIndex="0"
+            role="button"
+            onClick={this.onClickShowLessComment.bind(this)}
+            onKeyPress={this.onClickShowLessComment.bind(this)}
+          >
+            less comments
+          </span>
+          )}
+          {this.state.allFeedback.length > 5 && this.state.last < this.state.allFeedback.length
+          && (
+            <span
+              className="commSpan"
+              tabIndex="0"
+              role="button"
+              onClick={this.onClickShowMoreComment.bind(this)}
+              onKeyPress={this.onClickShowMoreComment.bind(this)}
+            >
+              more comments
+            </span>
+          )}
+        </div>
+      </Container>
+    );
+  }
+}
+
+Feedback.propTypes = {
+  id: PropTypes.string.isRequired,
```

Comment:
why there is a string, id must be a number?

Reply:
Try to launch this code with 'number'. It doesn't work. 

We need to pass props as a number. Everything I tried didn't work. Or may be I couldn't do that.